scripts/proc.sub_03_exp1

```diff
@@ -0,0 +1,512 @@
+#      (resampled from tlrc anat)
+3dresample -master full_mask.$subj+tlrc -input sub-03_ses-test_T1w_ns+tlrc \
+           -prefix rm.resam.anat
+
+# convert to binary anat mask; fill gaps and holes
+3dmask_tool -dilate_input 5 -5 -fill_holes -input rm.resam.anat+tlrc       \
+            -prefix mask_anat.$subj
+
+# compute overlaps between anat and EPI masks
+3dABoverlap -no_automask full_mask.$subj+tlrc mask_anat.$subj+tlrc         \
+            |& tee out.mask_ae_overlap.txt
+
+# note Dice coefficient of masks, as well
+3ddot -dodice full_mask.$subj+tlrc mask_anat.$subj+tlrc                    \
+      |& tee out.mask_ae_dice.txt
+
+# ---- create group anatomy mask, mask_group+tlrc ----
+#      (resampled from tlrc base anat, MNI_avg152T1+tlrc)
+3dresample -master full_mask.$subj+tlrc -prefix ./rm.resam.group           \
+           -input /Users/monica/abin/MNI_avg152T1+tlrc
+
+# convert to binary group mask; fill gaps and holes
+3dmask_tool -dilate_input 5 -5 -fill_holes -input rm.resam.group+tlrc      \
+            -prefix mask_group
+
+# ================================= scale ==================================
+# scale each voxel time series to have a mean of 100
+# (be sure no negatives creep in)
+# (subject to a range of [0,200])
+foreach run ( $runs )
+    3dTstat -prefix rm.mean_r$run pb03.$subj.r$run.blur+tlrc
+    3dcalc -a pb03.$subj.r$run.blur+tlrc -b rm.mean_r$run+tlrc \
+           -c mask_epi_extents+tlrc                            \
+           -expr 'c * min(200, a/b*100)*step(a)*step(b)'       \
+           -prefix pb04.$subj.r$run.scale
+end
+
+# ================================ regress =================================
+
+# compute de-meaned motion parameters (for use in regression)
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+           -demean -write motion_demean.1D
+
+# compute motion parameter derivatives (for use in regression)
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+           -derivative -demean -write motion_deriv.1D
+
+# create censor file motion_${subj}_censor.1D, for censoring motion 
+1d_tool.py -infile dfile_rall.1D -set_nruns 1                                \
+    -show_censor_count -censor_prev_TR                                       \
+    -censor_motion 0.5 motion_${subj}
+
+# note TRs that were not censored
+set ktrs = `1d_tool.py -infile motion_${subj}_censor.1D                      \
+                       -show_trs_uncensored encoded`
+
+# ------------------------------
+# run the regression analysis
+3dDeconvolve -input pb04.$subj.r*.scale+tlrc.HEAD                            \
+    -censor motion_${subj}_censor.1D                                         \
+    -polort 4                                                                \
+    -num_stimts 17                                                           \
+    -stim_times 1 stimuli/adj_detection_times.1D 'BLOCK(16.25)'              \
```

Comment:
Here is another major problem with your design matrix. Sorry I didn't catch this the first time. Although the presentation paradigm does use 16.25sec blocks, this analysis is modelling correct and incorrect responses separately so we need to use an event related design for the analysis. You can change `'BLOCK(16.25)'` to `'GAM'`.